src/lib/translators/mongo.ts

```diff
@@ -192,7 +192,7 @@ function transformPercentage(step: Readonly<S.PercentageStep>): MongoStep[] {
     { $unwind: '$_vqbAppArray' },
     {
       $project: {
-        [step.column]: {
+        [step.newColumnName ?? `${step.column}_PCT`]: {
```

Comment:
I don't know this syntax, what does it ?

Reply:
if `newColumnName ` exists, then `newColumnName ` else the other value

Inn computer language if you prefer, shortcut for:

`step.newColumnName !== undefined ? step.newColumnName : `${step.column}_PCT``